File: old/Way Too Long Words/1.java
import java.util.Scanner;
public class waytolongword {
	public static void main(String[] arg){
		Scanner in = new Scanner(System.in);
		int limit;
		String word;

		limit = in.nextInt();
		while (limit>0){
			word =in.next();
			char[] words = new char[100];
			words = word.toCharArray();
			if (word.length()>10){
				System.out.format("%s%d%s\n",words[0],word.length()-2,words[word.length()-1]);
			}
			else{
				System.out.format("%s\n",word);
			}
			limit--;
		}
	}
}

File: old/Way Too Long Words/1.c
#include <stdio.h>
#include <string.h>
int main(){
	int limit;
	char word[50];
	scanf("%d",&limit);
	while (limit>0){
		scanf("%s",&word);
		if(strlen(word)>10){
		printf("%c%d%c\n",word[0],strlen(word)-2,word[strlen(word)-1]);
		limit--;
		}
		else{
			printf ("%s\n",word);
			limit--;
		}
}
}